tests/unit/utils/test_get_marketplace_tags.py

```diff
@@ -7,16 +7,28 @@
 
 class TestGetMarketplaceTags(unittest.TestCase):
 
+  MOCK_MARKETPLACE_PRODUCTT_CODE_SC = {
```

Comment:
Very minor:  You have a typo in the variable name: MOCK_MARKETPLACE_PRODUCTT_CODE_SC -> MOCK_MARKETPLACE_PRODUCT_CODE_SC

Reply:
fixed